images/win/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -16,29 +17,34 @@ function Get-PostgreSQLMarkdown
         [PSCustomObject]@{ Property = "Path"; Value = $pgRoot },
         [PSCustomObject]@{ Property = "UserName"; Value = $env:PGUSER },
         [PSCustomObject]@{ Property = "Password"; Value = $env:PGPASSWORD }
-    ) | New-MDTable
+    )
 
-    Build-MarkdownElement -Head $name -Content $content
+    $headerNode.AddTable($content)

```

Comment:
What do you think about returning just a table from this function instead of node?
- We can rename function to `Get-PostgreSQLTable`
- Use the following construction in the Generator.ps1 to invoke the function:
```pwsh
$mainHeader.AddHeader("PostgreSQL").AddTable($(Get-PostgreSQLTable))
```

It could be a bit clearer.

Reply:
The same for MongoDB

Reply:
agree, corrected